backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/accounting.feature

```diff
@@ -132,7 +132,7 @@ Feature: accounting-override-feature
 
     And after not more than 30s, the matchInvoice document with identifier matchInv_1 has the following accounting records:
       | Fact_Acct_ID.Identifier | Account        | DR  | CR  | C_Currency_ID.Identifier | OPT.CurrencyRate | OPT.AccountConceptualName   |
-      | factAcct_4              | elementValue_4 | 0   | 0   | chf                      | 1                | NotInvoicedReceipts_Acct    |
+      | factAcct_4              | elementValue_4 | 0   | 0   | chf                      | 0                | NotInvoicedReceipts_Acct    |
```

Comment:
why change the currency rate from ONE to ZERO?

Reply:
this comes from this commit: https://github.com/metasfresh/metasfresh/commit/a68141f4aee9bf2d86e74bbcea7bec2502b26f30, more specifically from this method: `org.compiere.acct.FactLine#computeCurrencyRate`.  I just updated the cucumber here, because tests are failing on `adorable_badger_uat` and I thought it is a quick change